src/EED1E0.c

```diff
@@ -6,38 +6,15 @@
 #define NAMESPACE EED1E0
 
 #define gPartnerActionStatus dead_gPartnerActionStatus2
+#define entity_upgrade_block_hide_content dead_entity_upgrade_block_hide_content
 
-#include "world/common/SetOverrideFlags_40.inc.c"
+#define create_popup_menu dead_create_popup_menu
+#define hide_popup_menu dead_hide_popup_menu
+#define destroy_popup_menu dead_destroy_popup_menu
+#define heap_free dead_heap_free
+#define wPartnerHudScripts D_80100060
+#define wDisabledPartnerHudScripts D_801000A0
```

Comment:
These symbols should probably be renamed to give them the dead_ prefix. And also go in dead.h?

Reply:
Thanks, I did some of them and held off on the rest as I wasn't exactly sure how the dead stuff is supposed to be organized.